.gitattributes

```diff
@@ -0,0 +1,4 @@
+# See https://git-scm.com/docs/gitattributes for more about git attribute files.
+
+# Mark any vendored files as having been vendored.
```

Comment:
Did you mean:

```suggestion
# Mark any vendor files as having been vendored.
```

Reply:
I just reused the official Rails file 🙈 

https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/gitattributes.tt

Reply:
Said differently, I just removed the line with the schema file, to reduce any deviation with a fresh `rails new` project 👍